### PR TITLE
Fixed clang compilation issues

### PR DESCRIPTION
**Pull request by klingaard**
Unfortunately my editor removed a lot of superfluous white space which makes this PR look big.

### CODE REVIEW

**Comment by klingaard** [comment]
closes #128

**Comment by klingaard** [comment]
@Kunal-Buch, please take a look.  Tell GitHub to diff with no whitespace diffs.  Sorry about the formatting.  My editor removes dead whitespace (I need to update the Coding Style Guidelines).

**Review by arupc — APPROVED** [review]

File: .github/actions/build/entrypoint.sh [hunk 1]
```diff
@@ -12,6 +12,9 @@ echo "GITHUB_EVENT_PATH:" $GITHUB_EVENT_PATH
 echo "CONDA_PREFIX:" $CONDA_PREFIX
 echo "PWD:" `pwd`
 
+echo "Force clean of ccache"
```
Comment (h0lyalg0rithm) [inline .github/actions/build/entrypoint.sh:15]: @klingaard You can delete the caches from here https://github.com/riscv-software-src/riscv-perf-model/actions/caches

Comment (klingaard) [reply .github/actions/build/entrypoint.sh:15]: I know and I did.  Still have weird behavior.  Trying brute force to eliminate possibility.

**Pull request merged**